pytest-profiling/pytest_profiling.py

```diff
@@ -51,17 +53,85 @@ def pytest_sessionfinish(self, session, exitstatus):  # @UnusedVariable
             combined.dump_stats(self.combined)
             if self.svg:
                 self.svg_name = os.path.abspath(os.path.join(self.dir, "combined.svg"))
-                t = pipes.Template()
-                t.append("{} -f pstats $IN".format(self.gprof2dot), "f-")
-                t.append("dot -Tsvg -o $OUT", "-f")
-                t.copy(self.combined, self.svg_name)
+
+                # convert file <self.combined> into file <self.svg_name> using a pipe of gprof2dot | dot
+                # gprof2dot -f pstats prof/combined.prof | dot -Tsvg -o prof/combined.svg
+
+                # A/ Legacy: python pipes: does not seem to work on windows targets :(
+                # import pipes
+                # t = pipes.Template()
+                # t.append("{} -f pstats $IN".format(self.gprof2dot), "f-")
+                # t.append("dot -Tsvg -o $OUT", "-f")
+                # t.copy(self.combined, self.svg_name)
+
+                # B/ A handcrafted POpen pipe actually seems to work on windows:
+                from subprocess import Popen, PIPE
```

Comment:
*pep8* Could you move these imports to the top of the file, thanks

Reply:
sure, this was mostly to show the impact of both options on the imports